### ISSUE TITLE
process data

### ISSUE THREAD

**Issue by hu-hwb** [MCZhi/DTPP#16]
Error in us-nv-las-vegas-strip_0c380a4b55675374

It seems to be the error at 240 line in data_process.py

# get candidate trajectories
            try:
                first_stage_trajs, second_stage_trajs = self.get_ego_candidate_trajectories()
            except:
                print(f"Error in {map_name}_{token}")
                continue

**Comment by MCZhi** [comment]
The trajectory generator might fail in certain cases or the generated trajectories may deviate from the ground truth. So I have added this line to handle such situations, but it is expected to be ignored and not used in training.

**Comment by hu-hwb** [comment]
> The trajectory generator might fail in certain cases or the generated trajectories may deviate from the ground truth. So I have added this line to handle such situations, but it is expected to be ignored and not used in training.

Thank you for your prompt reply！

**Comment by hu-hwb** [comment]
However, another issue is that after data processing, there are no train and valid sub folders in the processed folder

**Comment by Jerome-Cong** [comment]
> However, another issue is that after data processing, there are no train and valid sub folders in the processed folder

Hi, how did you solve this?

**Comment by fanyu2021** [comment]
> However, another issue is that after data processing, there are no train and valid sub folders in the processed folder

I manually split this processed data into a training set and a validation set as the ratio 7:3.
